tasks/bootstrap-tasks.yml

```diff
@@ -62,6 +62,7 @@
     - htop
     - libgmp-dev
     - tmux
+    - tree
```

Comment:
@goetzc not sure if we need to hardcode all these packages. I think as a user of the role I should have the ability to decide whether to install all these packages or not. I propose to move this list to `defaults/main.yml`:

```YAML
sb_debian_packages_extra_packages:
  - ... 
  - tree
```

And then read that variable:

```YAML
- name: Install extra packages
  apt:
     name: "{{ item }}"
     state: present
  with_items: "{{ sb_debian_base_extra_packages }}"
```

That way users should be able to override the `sb_debian_base_extra_packages` list.

@00dav00, @lpaulmp, @jsl any comments?